tests/unit_tests/test_image_bounder.py

```diff
@@ -0,0 +1,67 @@
+from SyntheticDataset.image_operations import *
+import unittest
+from PIL import Image
+
+class ImageBounderTestCase(unittest.TestCase):
+
+
+    def test_getBoundsOfColor(self):
+        self.test_img1 = Image.open("tests/images/test_image2.png")
```

Comment:
Please rename the images to something more appropriate. The names should relate to the test case with which they are associated to.